### ISSUE TITLE
Heroku app offline... can you fix this?

### ISSUE THREAD

**Issue by x5nder** [lentz/youtube-shuffle#239]
Hey! I came upon this tool because I'm frustrated that the tvOS Youtube app _still_ can't shuffle playlists (I mean, c'mon, how long have users been asking for this?). Unfortunately, the Herokuapp link shows me: There's nothing here, yet.

Can you put the tool back online, pretty pretty please?

**Comment by lentz** [comment]
I've brought the app back online at https://youtube-shuffle.fly.dev
Assuming usage doesn't exceed the free tier, I can keep it running there.

**Comment by x5nder** [comment]
This app is blocked
This app tried to access sensitive info in your Google Account. To keep your account safe, Google blocked this access.

Google definitely doesn't like it ;-)

**Comment by lentz** [comment]
Yeah, that's just because I haven't gone through their verification process. You should still be able to click through to the site.

**Comment by x5nder** [comment]
Nope... Chrome (and Safari) don't show any advanced link or pop-up anymore... I think Google changed that recently. Annoying... :x

**Comment by lentz** [comment]
Can you attach a screenshot?

**Comment by x5nder** [comment]
Sorry, slow response... yeah, sure!

![Screenshot 2023-10-27 at 11 00 56](https://github.com/lentz/youtube-shuffle/assets/4243542/61b7afa2-3c13-4ef0-83bc-f6f3e4affdb8)

**Comment by lentz** [comment]
Ok, unfortunately it looks like I'll have to go through Google's
verification process even for this super basic functionality.
